### PR TITLE
fixes #322 - added script to expose apps on loklak app wall

### LINKED ISSUE #322: Add script to expose app on Loklak app wall
Add a script to automatically expose app on loklak app wall so that there is no need to copy and paste individual app.json file in the root apps.json file.

### PR DESCRIPTION
**Pull request by djmgit**
fixes issue #322, added script to automatically expose
app on Loklak app wall.

I have:
- [x] There is a corresponding issue for this pull request.
- [x] Mentioned the Issue number in the pull request commit message `Fixes #<number> commit message`
- [x] There is only strictly only one commit per issue.

### For the reviewers
I have:
- [ ] Reviewed this pull request by an authorized contributor.
- [ ] The reviewer is assigned to the pull request.

### CODE REVIEW

**Comment by djmgit** [comment]
@kavithaenair @singhpratyush @Achint08 @@SKrPl  open for review

**Comment by djmgit** [comment]
expose

File: bin/updatewall.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
+
+import json
+from collections import OrderedDict
+
+PATH_TO_ROOT_JSON = "../apps.json"
```
Comment (vibhcool) [inline bin/updatewall.py]: generally in python, single quotes are recommended

**Comment by djmgit** [comment]
@vibhcool removed double quotes. Please re-review.

**Pull request merged**